tests/ignite/metrics/test_classification_report.py

```diff
@@ -77,24 +80,22 @@ def update(engine, i):
 def _test_integration_multilabel(device, output_dict):
 
     rank = idist.get_rank()
-    torch.manual_seed(12)
 
     def _test(metric_device, n_epochs, labels=None):
 
         classification_report = ClassificationReport(device=metric_device, output_dict=output_dict, is_multilabel=True)
 
         n_iters = 10
-        s = 16
+        batch_size = 16
         n_classes = 7
 
-        offset = n_iters * s
-        y_true = torch.randint(0, 2, size=(offset * idist.get_world_size(), n_classes, 6, 8)).to(device)
-        y_preds = torch.randint(0, 2, size=(offset * idist.get_world_size(), n_classes, 6, 8)).to(device)
+        y_true = torch.randint(0, 2, size=(n_iters * batch_size, n_classes, 6, 8)).to(device)
+        y_preds = torch.randint(0, 2, size=(n_iters * batch_size, n_classes, 6, 8)).to(device)
 
         def update(engine, i):
             return (
-                y_preds[i * s + rank * offset : (i + 1) * s + rank * offset, ...],
-                y_true[i * s + rank * offset : (i + 1) * s + rank * offset, ...],
+                y_preds[i * batch_size : (i + 1) * batch_size],
```

Comment:
@puhuk why do you suppress `...` here ?

Reply:
I thought the output is same. Let me check and correct this.